src/errors.ts

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable max-len */
+const errors: {[key: string]: ({ message } : { message?: string }) => string} = {
```

Comment:
Cannot named error take enum? It would be way easier to validate that we have correct entries

Reply:
Refactored